src/service.rs

```diff
@@ -15,7 +17,9 @@ pub struct ServiceProperties {
     pub name: String,
     pub description: String,
     pub http_url: String,
-    pub ws_url: String
+    pub ws_url: String,
+    #[serde(rename="properties")]
```

Comment:
per request from gmarty, but it doesn't really matter since we change format soon anyway, so no preference for me.